### PR TITLE
fix(opetorhub): adjusts publishing process to repository split

### LINKED ISSUE #893: Update OperatorHub release process after repository split
##### Issue Overview

The OperatorHub has split in two different repositories between upstream and Red Hat.
See README https://github.com/operator-framework/community-operators

##### Expected Behaviour

Releasing of Operator to Catalog.

##### Current Behaviour

```
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.        
remote: 
remote: Create a pull request for 'community-operators/istio-workspace-operator-0.4.0' on GitHub by visiting:        
remote:      https://github.com/maistra/community-operators/pull/new/community-operators/istio-workspace-operator-0.4.0        
remote: 
To https://github.com/maistra/community-operators.git
 * [new branch]        community-operators/istio-workspace-operator-0.4.0 -> community-operators/istio-workspace-operator-0.4.0
{
  "message": "Repository was archived so is read-only.",
  "documentation_url": "https://docs.github.com/rest/reference/pulls#create-a-pull-request"
}
CircleCI received exit code 0
```

### PR DESCRIPTION
**Pull request by bartoszmajsak**
#### Short description of what this resolves:

Operator Hub repository has been split to two repos:

* operator-framework/community-operators/community-operator --> redhat-openshift-ecosystem/community-operators-prod/operators
* operator-framework/community-operators/upstream-community-operators --> k8s-operatorhub/community-operators/operators

This PR addresses this change and fixes release automation.

#### Changes proposed in this pull request:

- split between test and publish (separated scripts)
- clean-up in the scripts
- updates the release job for CircleCI

Fixes #893

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
@aslakknutsen we need to review the template as I think we should tick more checkboxes.

**Comment by bartoszmajsak** [comment]
When we merge it we should release operator `0.4.0` again :)

File: scripts/release/operatorhub/test.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        exit 1
+        break
+        ;;
+  esac
+done
+
+CUR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+BUNDLE_DIR="${CUR_DIR}"/../../../bundle
+
+GITHUB_TOKEN="${GITHUB_TOKEN:-}"
```
Comment (aslakknutsen) [inline scripts/release/operatorhub/test.sh:48]: These are all the same variables arn't they? Source them?

**Comment by aslakknutsen** [comment]
The Subject of this PR sounds a bit off?

File: scripts/release/operatorhub/test.sh (hunk 1 above)
Comment (bartoszmajsak) [reply scripts/release/operatorhub/test.sh:48]: As in have a shared file between `publish` scripts? Not sure we gain much but we make it a bit more difficult to navigate.

**Comment by bartoszmajsak** [comment]
> The Subject of this PR sounds a bit off?

yeah, it took the first commit msg. fixed. thanks.

File: scripts/release/operatorhub/test.sh (hunk 1 above)
Comment (aslakknutsen) [reply scripts/release/operatorhub/test.sh:48]: maybe.. ok

**Review by aslakknutsen — APPROVED** [review]

**Pull request merged**